src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
+  margin: 0 auto;
+
+  max-width: 1200px;
+  min-width: 800px;
+  --blue-color: #00acdc;
+  --header-height: 23px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__link {
+  max-height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  margin: var(--header-height) 0;
+  padding-left: 0;
+  font-size: 0;
+}
+
+.nav__item {
+  display: inline-block;
   margin: 0;
+  padding: 0;
+  align-items: center;
+  height: fit-content;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  transition: 0.2s;
+  padding: var(--header-height) 0;
+  height: 60px;
+}
+
+.nav__link:hover {
+  transition: 0.2s;
+  color: var(--blue-color);
+}
+
+.is-active {
+  color: var(--blue-color);
+  position: relative;
```

Comment:
Don't alter position based on modifier - better move `position: relative;` to nav__link